lib/core/ring/cne_ring_api_internal.h

```diff
@@ -20,6 +20,8 @@
 extern "C" {
 #endif
 
+#define RING_F_ALLOCATED 0x8000 /**< The ring structure and data was allocated */
```

Comment:
I know this flag is only used for internal alloc/free, but should we move it to cne_ring_api.h where the other flags are defined? Somebody might accidentally add another flag and choose this same value without realizing its already in use. Also, we provide a cne_ring_get_flags() which would return the entire flags, which might confuse applications to see undefined flags set.

Reply:
Originally I did have it in the cne_ring_api.h file :-( moved it back.